src/migrations/20241108102623-GrantRelationshipToActive.js

```diff
@@ -0,0 +1,46 @@
+const { prepMigration } = require('../lib/migration');
+
+const goalText = 'The recipient will develop and implement a CAP/QIP to address monitoring findings.';
```

Comment:
Its missing the prefix, something like "(Monitoring)"

Reply:
Sure it wasn't in the ticket, but I will add it.